### ISSUE TITLE
Change scripts

### ISSUE THREAD

**Issue by DaniGuardiola** [mixmaxhq/deyarn#4]
Some scripts can call other scripts via yarn.

For example, "docs": "yarn -s code-doc && yarn -s api-doc".

I guess the possible flags can complicate it a bit. The -s flag is for silent.

**Comment by wearhere** [comment]
Yeah I'm not sure if we'd want to automatically update them. Rather, the script currently suggests that the user updates the scripts and prints them out to see if that's needed: https://github.com/mixmaxhq/deyarn/blob/24b36eda382211e066803b30bc6ef8a4ad2fca94/src/index.js#L93-L94. We'd look at a PR though!
